test/TestMemoryFS.test.js

```diff
@@ -0,0 +1,63 @@
+import path from 'path';
```

Comment:
Please add test via a separate PR, looks good and is appreciated but we normally do it a bit different

Reply:
Yep, was thinking the same 👍 Did it here as wanted to refactor the `./examples` to align overall structure... Will drop it from here